### PR TITLE
Upload data to `results-dashboard-data` ES

### PR DESCRIPTION
**Pull request by pablomh**

### CODE REVIEW

File: experiment/run-library.sh [hunk 1]
```diff
@@ -167,9 +168,17 @@ function status_data_create() {
...
+        rdd_file=${RDD_FILE}
+    else
+        rdd_file="${sd_log}.rdd.json"
+        rm -f "${rdd_file}"
+    fi
     if [ -n "$PARAM_inventory" ]; then
         sd_hostname="$( ansible $opts_adhoc --list-hosts satellite6 2>/dev/null | tail -n 1 | sed -e 's/^\s\+//' -e 's/\s\+$//' )"
     fi
+    workdir_url="${PARAM_workdir_url:-https://workdir-exporter.example.com/workspace}"
+    sd_link="${workdir_url}/${JOB_NAME:-NA}/${sd_log}"
```
Comment (jhutar) [inline experiment/run-library.sh:181]: Once the test finishes, we move the files to `.../$sd_run/`, is that part of `$sd_log`?

File: experiment/run-library.sh [hunk 2]
```diff
@@ -248,6 +258,32 @@ function status_data_create() {
       python -c "import sys, json; obj, pos = json.JSONDecoder().raw_decode(sys.stdin.read()); assert '_shards' in obj and  obj['_shards']['successful'] >= 1 and obj['_shards']['failed'] == 0, 'Failed to upload status data: %s' % obj"
     ###status_data.py --status-data-file $sd_file --info
 
+    # Create "results-dashboard-data" data file
+    set -x
+    status_data.py \
```
Comment (jhutar) [inline experiment/run-library.sh:263]: This creates unneeded fields/structure in the JSON.

Do we have `jq` installed? Maybe use something like this:

```
$ echo '{}' | jq '.group = "Core Platform" | .product = "Red Hat Satellite" | .version = "1.2.3"'
{
  "group": "Core Platform",
  "product": "Red Hat Satellite",
  "version": "1.2.3"
}
```

**Review by jhutar — APPROVED** [review]
Thank you!

File: experiment/run-library.sh (hunk 2 above)
Comment (pablomh) [reply experiment/run-library.sh:263]: I didn't use it because I didn't know it was that easy :) Will amend it in a follow-up commit.

**Pull request merged**